src/tasks/pair-liquidity-info-history-importer/pair-liquidity-info-history-importer.service.ts

```diff
@@ -1,19 +1,19 @@
 import { Injectable, Logger } from '@nestjs/common';
-import { MdwHttpClientService } from '../clients/mdw-http-client.service';
+import { MdwHttpClientService } from '../../clients/mdw-http-client.service';
```

Comment:
should we introduce some sort of absolute paths?

https://stackoverflow.com/questions/57439410/how-nestjs-configures-path-aliases-in-a-project

Reply:
Good idea, I'll introduce this in a different PR